x-pack/plugins/alerting/public/pages/maintenance_windows/translations.ts

```diff
@@ -211,15 +211,15 @@ export const CREATE_FORM_CATEGORY_STACK_RULES = i18n.translate(
 export const CREATE_FORM_SCOPED_QUERY_TITLE = i18n.translate(
   'xpack.alerting.maintenanceWindows.createForm.scopedQuery.title',
   {
-    defaultMessage: 'Define scope of rules',
+    defaultMessage: 'Scope of rules',
```

Comment:
Do we have to have this title? I don't think it adds a lot of value.

If we have to have a title, IMO "scope of rules" incorrectly implies we're limiting the rules affected by this MW, rather than the alerts. Other options:
- "Scope of maintenance window" (IMO too generic since all these other timeframe/category options limit its scope too)
- "Alert filters" / "Filter alerts" (Just repeats the switch label, so not helpful)
- "Filters" ( potentially confusing since this section just enables filtering but the actual filters appear in subsequent section)

So my vote would be for removing that title entirely.

Reply:
I think it might look odd for scope query to be the only section without a title? 

![image](https://github.com/elastic/kibana/assets/74562234/09d067f4-d0a6-45b5-a83c-cfa75f34ce26)

Reply:
I personally think it would be fine, but if a title is required my vote would be for "Filters"

Reply:
I decided to keep the label since we are following the EUI described form blocks format: https://eui.elastic.co/#/forms/form-layouts#described-form-groups